lib/core/base/context.js

```diff
@@ -128,7 +128,7 @@ function parseSelectorArray(context, type) {
 		if (typeof item === 'string') {
 			result = result.concat(utils.toArray(document.querySelectorAll(item)));
 			break;
-		} else if (item && item.length) {
+		} else if (item && Array.isArray(item)) {
```

Comment:
Not a maintainer over here, but `Array.isArray` may be too new for the desired browser support of axe-core. I'd probably test it this way: `Object.prototype.toString.call(item) === '[object Array]';`